proto/common.proto

```diff
@@ -15,6 +15,8 @@ enum ChainRef {
     CHAIN_ETHEREUM = 100;
     CHAIN_ETHEREUM_CLASSIC = 101;
 
+    CHAIN_POLYGON = 137;
```

Comment:
[`proto/common.proto` in emeraldpay/emerald-grpc](https://github.com/emeraldpay/emerald-grpc/blob/master/proto/common.proto) seems to use id 1002 instead:

```
    // Sidechains and state channels start with 1_000
    [...]
    CHAIN_MATIC = 1002; // Matic PoS Ethereum sidechain based on Polygon
```

Reply:
I assume this is the maintainer trying to implement support outside of this PR. This PR can be updated or closed once his work is complete.